.github/workflows/deploy.yml

```
@@ -13,6 +13,8 @@ jobs:
         persist-credentials: false
     - name: Install SSH Client
       uses: webfactory/ssh-agent@v0.2.0
+      env:
+        key: ACTIONS_ALLOW_UNSECURE_COMMANDS=true
```

Try a newer version instead of allowing unsecure commands
```suggestion
      uses: webfactory/ssh-agent@v0.4.1
```